### ISSUE TITLE
Options are ignored

### ISSUE THREAD

**Issue by sgen** [caseyWebb/generator-npm-init#27]
Running this generator: 
`'use strict';

var generators = require('yeoman-generator');

module.exports = generators.Base.extend({
  initNPM: function() {
    this.composeWith('npm-init', {
      // skip prompts
      'skip-name': false,
      'skip-description': false,
      'skip-version': false,
      'skip-main': false,
      'skip-test': false,
      'skip-repo': false,
      'skip-keywords': false,
      'skip-author': false,
      'skip-license': false,

      // supply alternative defaults
      name: '<%= destFolderName %>',
      version: '1.0.0',
      description: '',
      main: 'index.js',
      test: 'echo "Error: no test specified" && exit 1',
      repo: '',
      keywords: [],
      author: '',
      license: 'ISC'
    }, {
      local: require.resolve('generator-npm-init/app')
    });
  }
});`

the user promped for name, description, version, etc. and the defaults are ignored

**Comment by sgen** [comment]
After adding a console.log(options) to initializing() and prompting() I get the following (relevant) output in each:

`...
  'skip-cache': false,
  'skip-install': false,
  'skip-name': undefined,
  'skip-description': undefined,
  'skip-version': undefined,
  'skip-main': undefined,
  'skip-test': undefined,
  'skip-repo': undefined,
  'skip-keywords': undefined,
  'skip-author': undefined,
  'skip-license': undefined,
  name: undefined,
  description: undefined,
  version: undefined,
  main: undefined,
  test: undefined,
  repo: undefined,
  keywords: undefined,
  author: undefined,
  license: undefined }
`

it seems that the options object is not being passed to the generator?

**Comment by caseyWebb** [comment]
did you mean to close this?

**Comment by sgen** [comment]
Yes I did. I miss read the Yeoman.js docs and passed `{ ... }` to `composeWith()` instead of `{ options: { ... } }`.

You might want to expand the docs to include an example using the options to safeguard against documentation skimming Yeoman noobs
